src/Modules/getMeal.js

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable */
+import { addLike } from "./userApi.js";
+import mealsFunction from "./display.js";
+/* eslint-enable */
+const modalDetailsContent = document.querySelector('.modal-details-content');
+const recipeCloseBtn = document.getElementById('modal-close-btn');
+
+recipeCloseBtn.addEventListener('click', () => {
+  modalDetailsContent.parentElement.classList.remove('displayModal');
+});
+
+function getMealInfo(e) {
+  e.preventDefault();
+
+  if (/like-/.test(e.target.id)) {
+    const { id } = e.target;
+    const foodNum = id.match(/\d+$/)[0];
+
+    addLike(foodNum);
+  }
+
+  if (e.target.classList.contains('comment-btn')) {
+    const mealItem = e.target.parentElement.parentElement;
+    fetch(
+      `https://www.themealdb.com/api/json/v1/1/categories.php?i=${mealItem.dataset.id}`,
+    )
+      .then((response) => response.json())
+      .then((data) => mealsFunction(data.categories));
+  }
+}
```

Comment:
Just so you know, when I try to get more information about a meal, I keep getting information about the first food on the list. Please take a look at this; users should be able to see the information on the specific meal they chose. 👍🏽

Reply:
The License link is fixed